src/sentry/signals.py

```diff
@@ -100,6 +100,7 @@ def send_robust(self, sender, **named):
 buffer_incr_complete = BetterSignal(providing_args=["model", "columns", "extra", "result"])
 pending_delete = BetterSignal(providing_args=["instance", "actor"])
 event_processed = BetterSignal(providing_args=["project", "event"])
+setup_subscription = BetterSignal(providing_args=["organization", "user"])
```

Comment:
Should this be called `create_organization`? 🤷

Reply:
Yeah agree, signals should be named based on the thing that happened rather than what you want to do with them.

Reply:
Yeah, how's `org_setup_complete`? Subscriptions require an organization member to be created, so I wanted to be clearer why it's not a post_save on organization.

Reply:
I think `org_setup_complete` is great.